docs/pages/web/sheet.js

```diff
@@ -261,6 +273,221 @@ Sheet comes in 3 sizes: small (\`sm\`), medium (\`md\`), and large (\`lg\`).
             }
           />
         </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Dismiss confirmation"
+          description={`There are two ways Sheet can be dismissed: internally-controlled and externally-controlled dismiss actions.
+
+The three internally-controlled or component-controlled dismiss actions are:
+- when the \`ESC\` key is pressed
+- when the backdrop is clicked
+- when the dismiss IconButtons are clicked
+
+The externally-controlled dismiss actions require implementing the callback \`onDismissStart\`. See the [animation variant](#Animation) to learn more.
+
+Sheets can contain forms or be part of flows where the user is required to submit infomation. If a Sheet is dismissed involuntarily, the data entered by the user could not be saved and lost. This can create a bad user experience.
+
+To prevent dismissing Sheet involuntary, we can use \`dismissConfirmation\`. When provided, it will open a confirmation modal each time component-controlled dismiss actions are triggered.
+
+The confirmation modal has a flexible API. When the \`dismissConfirmation\` prop is set to "true", Sheet uses defauld texts and labels.
```

Comment:
typo: "…_default_ texts and labels"

Reply:
Also: I would end that last sentence with a `:`, since it's followed by a list of the various texts and labels

Reply:
DONE